+transformer/+layer/attention.m

```diff
@@ -63,16 +63,17 @@
 
 % Use the past
 if ~isempty(past)
-    PK = past(:,:,:,1);
-    PV = past(:,:,:,2);
+    PK = permute(past(:,:,:,1,:), [1 2 3 5 4]);
```

Comment:
Might be worth adding a comment reminding what the current dimension format is.

Reply:
Yes good idea -- I'll add a comment